db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 20231003000823494) do
+ActiveRecord::Schema[7.1].define(version: 20231003000823494) do
```

Comment:
This is not related to the sanitizer. Does that  happen when you run db:migrate?